website/pages/chapters/assessing_performance.js

```diff
@@ -193,22 +237,37 @@ export default function AssessingPerformance() {
           dealing with the <IM math={`(x, y)`} /> pairs. So then selecting the
           model that generalizes best, would then mean choosing the one with the
           lowest true error.
+          <MatthewComment>
+            I don't understand how these last two sentences are related to the
+            excpectation. It seems you've given the definition of the
+            expectation, and then said this looks like an expectation??
+          </MatthewComment>
         </p>
 
+        <h3> (Matthew:) Exiting theory land, back to practical ML</h3>
         <p>
           In many real-life circumstances, it turns out to not be possible to
-          compute this true error. You might not know the exact distributions of
-          the houses and their prices of all possible houses you could see in
-          the future! So without access to all future data, how can we actually
-          compute the true error?
-        </p>
-
-        <p>
-          A very common technique in machine learning suggests that if you
-          aren't able to exactly compute something, you can try to estimate it.
-          That's what we will do here. The basic idea is to hide part of our
-          dataset from our ML algorithm and use that hidden data as a proxy for
-          "all future data" after the predictor is trained.
+          compute this true error. You <strike>might not</strike> won't know the
+          exact distributions of the houses and their prices of all possible
+          houses you could see in the future! So without access to all future
+          data, how do we actually compute estimate the true error? A very
+          common
+          <MatthewComment>
+            Would you call this a "common technique" I feel like it's not a
+            technique, but I don't know what to call it?
+          </MatthewComment>
+          technique in machine learning suggests that if you aren't able to
+          exactly compute something, you can try to estimate it. That's what we
+          will do here. This amounts to <em>hiding</em> part of our dataset from
+          our ML algorithm and usng that hidden data as a proxy for "all future
+          data" when evaluating the predictor.
+          <MatthewComment>
+            Perhaps you should have an internal (to us) spreadsheet of names, i
+            feel like i've been using model/predictor/etc. to mean the same
+            things interchangably, and it'd be nice to centralize and standarize
+            that. You could also use this as a specification for when to bold
+            and when to emphasize
+          </MatthewComment>
```

Comment:
That's a good idea! I'll put something together. I also don't think you should worry **too much** about it since I will always go through and clean everything up! 

The biggest one is by far the model/predictor terminology. So the annoying thing is that people overload the term "model" so I tried to use different terminology to clarify the difference. Some people use "model" to describe the underlying phenomena (like the linear regression model). Other people use the term to describe the artifact trained from an ML algorithm. From the statistical learning perspective, people call this the "best hypothesis", but I use the term "predictor" to sound less scary :P